GetFed/GetFed/Controllers/FoodSearchViewController.swift

```diff
@@ -9,7 +9,11 @@
 import UIKit
 
 class FoodSearchViewController: UIViewController {
-
+    
+    // MARK - Outlets
+    
+    @IBOutlet weak var foodTableView: UITableView!
```

Comment:
`weak` isn't strictly necessary for `IBOutlet`s.

Reply:
I read up on this more (ex: https://cocoacasts.com/should-outlets-be-weak-or-strong) and I agree with you. I'm going to lean more towards strong references for IBOutlets from now on. From my reading, it seems like crashes can be avoided by doing this.